pkg/scheduler/framework/plugins/imagelocality/image_locality_test.go

```diff
@@ -236,11 +257,12 @@ func TestImageLocalityPriority(t *testing.T) {
 	nodeWithNoImages := v1.NodeStatus{}
 
 	tests := []struct {
-		pod          *v1.Pod
-		pods         []*v1.Pod
-		nodes        []*v1.Node
-		expectedList framework.NodeScoreList
-		name         string
+		featureEnabled bool
```

Comment:
No need to disable this featuregate when testing cases without imageVolume. Those cases should be tested to verify they works well with this fg enabled.
I think it should be ok to just enable this fg for all cases.

Reply:
Wouldn't we be missing the test case that verifies everything works as expected when the feature gate is disabled?

Reply:
For some test we run cases twice with fg enabled/disabled to ensure this. Like this one https://github.com/kubernetes/kubernetes/blob/39c640fe9051b60bacbacf47f64cb697fb807861/pkg/scheduler/backend/queue/scheduling_queue_test.go#L1066

But since the influence of fg enablement is rather small here. I think it should be fine.